project_proposal/stakeholders.md

```diff
@@ -0,0 +1,9 @@
+### Stakeholders
+
```

Comment:
I'm not sure, but I think we should avoid using personal pronouns as well.

Reply:
Personal pronouns, Kjako19?

Reply:
I, you, us, we, etc.